### PR TITLE
Add Logistic Regression Algorithm #35

### PR DESCRIPTION
**Pull request by hemanth-kotagiri**
### Using Scikit-Learn, implemented a Logistic Regression Model trained on MNIST Dataset.
Code includes the following:
- Feature scaling on the Dataset.
- Logistic Regression Model with an Accuracy Score of 89 %

### CODE REVIEW

**Comment by hemanth-kotagiri** [comment]
Could you please review this PR @hritik5102.

**Comment by hritik5102** [comment]
> Could you please review this PR @hritik5102.

Please add your code to the "neural network" folder and also document your code. 
Thank you

**Comment by hemanth-kotagiri** [comment]
@hritik5102 Thank you for your feedback. I will do the same!

**Comment by hemanth-kotagiri** [comment]
Hey there, @hritik5102 . Just committed new changes. Check'em out. Thanks!

**Comment by hritik5102** [comment]
> Hey there, @hritik5102 . Just committed new changes. Check'em out. Thanks!

Great, thanks for your contribution :)

**Comment by hemanth-kotagiri** [comment]
@hritik5102 Awesome man! I will contribute more and will soon raise PRs for new stuff. I'll close the issue related to this PR as well. Thanks!
